### PR TITLE
updated some news links

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/welcome.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-ms.date: 08/14/2017
+ms.date: 04/30/2018
 ms.topic: article
-ms.prod: .net
+ms.prod: dotnet
 ms.devlang: dotnet
-ms.assetid: cb788dcf-2120-467f-9c34-c02a90e1f68f
+ms.custom: "updateeachrelease"
 ---
 # Welcome to .NET
```
Comment (rpetrusha) [inline docs/welcome.md]: nit unrelated to your PR for line 14: The title is "Get Started with .NET Core", not "Getting...."

File: docs/welcome.md [hunk 2]
```diff
@@ -51,7 +48,7 @@ Additionally, you can browse the [.NET API reference](/dotnet/api).
```
Comment (rpetrusha) [inline docs/welcome.md]: The link to the .API reference on line 47 is broken.

File: docs/welcome.md [hunk 3]
```diff
@@ -18,23 +17,21 @@ Build many types of apps with .NET, such as cloud, IoT, and games using free cro
...
 - [Announcing Visual Studio 2017 General Availability](https://blogs.msdn.microsoft.com/visualstudio/2017/03/07/announcing-visual-studio-2017-general-availability-and-more/)
 - [What's new for .NET Core and Visual Studio 2017 (video)](https://channel9.msdn.com/events/Visual-Studio/Visual-Studio-2017-Launch/T108)
-- [Announcing the .NET Framework 4.7](https://blogs.msdn.microsoft.com/dotnet/2017/04/05/announcing-the-net-framework-4-7/)
-- [New Features in C# 7.0](https://blogs.msdn.microsoft.com/dotnet/2017/03/09/new-features-in-c-7-0/)
 - [Announcing F# 4.1 and the Visual F# Tools for Visual Studio 2017](https://blogs.msdn.microsoft.com/dotnet/2017/03/07/announcing-f-4-1-and-the-visual-f-tools-for-visual-studio-2017-2/)
-- [Open Source Xamarin, Ready for you!](https://blog.xamarin.com/live-from-evolve-open-source-xamarin-ready-for-you/)
-- [The week in .NET](https://blogs.msdn.microsoft.com/dotnet/tag/week-in-net/)
-- [Build 2017 on Channel 9 - Video on Microsoft's latest technologies and news!](https://channel9.msdn.com/?wt.mc_id=build_hp#programGuide)
 
 ## Documentation
```
Comment (rpetrusha) [inline docs/welcome.md]: For the links to the .NET documentation, given that they contain extensions, etc., might not links to docs be a better experience.

**Review by rpetrusha — APPROVED** [review]
I've left several comments unrelated to your PR, @mairaw. You can decide whether or not you want to address them.

File: docs/welcome.md (hunk 2 above)
Comment (mairaw) [reply docs/welcome.md]: I haven't changed that link and it works for me.

**Pull request merged**